int32 bearing_id

int32 edge_id

float64 edge_angle_diff

int32 visited
